### PR TITLE
Implement sign up validation tests with XCUITest

### PR DESCRIPTION
**Pull request by germanStabile**
Remove UI Testing done with KIF.

ezgif com-video-to-gif-13

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -17,9 +17,3 @@ target 'ios-base' do
   pod 'FBSDKLoginKit', '~> 5.5.0'
   # ------
 end
-
-target 'AcceptanceTests' do
-  pod 'KIF', '~> 3.7.4', :configurations => ['Debug']
-  pod 'KIF/IdentifierTests', '~> 3.7.3', :configurations => ['Debug']
-  pod 'OHHTTPStubs/Swift', '~> 8.0.0', :configurations => ['Debug']
-end
```
Comment (pMalvasio) [inline Podfile]: 👏

File: ios-baseUITests/XCUIApplicationExtension.swift [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+//  Copyright © 2020 Rootstrap. All rights reserved.
+//
+
+import XCTest
+
+extension XCUIApplication {
+  func type(text: String, on fieldName: String, isSecure: Bool = false) {
+    let fields = isSecure ? secureTextFields : textFields
+    let field = fields[fieldName]
+    field.forceTap()
```
Comment (pMalvasio) [inline ios-baseUITests/XCUIApplicationExtension.swift]: Why `forceTap()` here and for the clear method below just `tap()`?

File: ios-baseUITests/XCUIElementExtension.swift [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+//
+//  Created by Germán Stábile on 2/13/20.
+//  Copyright © 2020 TopTier labs. All rights reserved.
+//
+
+import XCTest
+
+extension XCUIElement {
+  
+  func swipeAround() {
```
Comment (glm4) [inline ios-baseUITests/XCUIElementExtension.swift]: Remove if not used.

File: ios-baseUITests/XCUIApplicationExtension.swift (hunk 2 above)
Comment (germanStabile) [reply ios-baseUITests/XCUIApplicationExtension.swift]: I think tap here is enough, I was having a weird issue (still occurring in iOS 13) and was trying things, anyways I think forceTap can be the default so I'll update clear.

**Pull request merged**